libs/vm_monitor/src/lib.rs

```diff
@@ -39,6 +39,15 @@ pub struct Args {
     #[arg(short, long)]
     pub pgconnstr: Option<String>,
 
+    /// Flag to signal that the Postgres file cache is not in memory, and therefore should not
+    /// count against available memory.
+    //
+    // NB: Making this flag positive (i.e. "file cache in memory", instead of "... not in memory")
+    // would have been easier to deal with, but in order to be backwards compatible during the
+    // switch away from an in-memory file cache, we had to default to the previous behavior.
+    #[arg(long)]
+    pub file_cache_not_in_memory: bool,
```

Comment:
Make it 'file-cache-on-disk`, perhaps?

Reply:
Fair enough - I think there was _some_ reason why I didn't want to do that originally, but I can't remember it now :sweat_smile: